### PR TITLE
core/deps: lmdb@2.5.2->2.7.11

### PR DESCRIPTION
**Pull request by legobeat**
# ↪️ Pull Request

## ✔️ PR Todo

- [ ] Added/updated unit tests for this change
- [ ] Filled out test instructions (In case there aren't any unit tests)
- [ ] Included links to related issues/PRs

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -9626,10 +9633,15 @@ node-forge@^1.2.1:
...
-  version "5.0.3"
-  resolved "https://registry.yarnpkg.com/node-gyp-build-optional-packages/-/node-gyp-build-optional-packages-5.0.3.tgz#92a89d400352c44ad3975010368072b41ad66c17"
-  integrity sha512-k75jcVzk5wnnc/FMxsf4udAoTEUv2jY3ycfdSd3yWu6Cnd1oee6/CfZJApyscA4FJOmdoixWwiwOyf16RzD5JA==
+node-gyp-build-optional-packages@5.0.6:
+  version "5.0.6"
+  resolved "https://registry.yarnpkg.com/node-gyp-build-optional-packages/-/node-gyp-build-optional-packages-5.0.6.tgz#2949f5cc7dace3ac470fa2ff1a37456907120a1d"
+  integrity sha512-2ZJErHG4du9G3/8IWl/l9Bp5BBFy63rno5GVmjQijvTuUZKsl6g8RB4KH/x3NLcV5ZBb4GsXmAuTYr6dRml3Gw==
+
+node-gyp-build-optional-packages@5.0.7:
+  version "5.0.7"
```
Comment (legobeat) [inline yarn.lock:9556]: I don't seem to be able to locate the sources of this package outside of the npm module in the registry.
This goes for the existing `5.0.3` as well as the new ones introduced here.

Comment (legobeat) [reply yarn.lock:9556]: https://github.com/kriszyp/node-gyp-build/tree/v5.0.7

**Comment by legobeat** [comment]
Windows error in integration tests

https://github.com/nodejs/node/blob/842e5a10ac59c826f875ef79e92d144e2655cc9a/test/common.js#L502-L505

```
 hmr
    hmr server
error Command failed with exit code 3221225477.
      √ should emit an HMR update for the file that changed (423ms)
      √ should emit an HMR update for all new dependencies along with the changed file (219ms)
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 3221225477.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 3221225477
Command: C:\hostedtoolcache\windows\node\14.21.3\x64\node.exe
Arguments: C:\npm\prefix\node_modules\yarn\lib\cli.js test-ci
Directory: D:\a\parcel\parcel\packages\core\integration-tests
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspace for documentation about this command.
error Command failed with exit code 3221225477.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

**Comment by mischnic** [comment]
You can ignore these, Windows CI is pretty flakey

**Comment by legobeat** [comment]
Failing Benchmarks workflow addressed in https://github.com/parcel-bundler/parcel-benchmark-action/pull/93.

**Review by devongovett — APPROVED** [review]

**Pull request merged**